kubernetes/jobs/migrate-database.yml.j2

```diff
@@ -32,3 +37,19 @@ spec:
             - name:       database-job-settings
               mountPath:  /srv/http/concent_api/concent_api/settings/config_map/
               readOnly:   true
+        {% if initialize_django_admin %}
+        - name:             create-django-admin
+          image:            {{ docker_registry }}/{{ image_prefix }}concent-api:{{ image_version }}
+          imagePullPolicy:  Always
```

Comment:
This script is not supposed to load the admin fixture. It's not safe because the database you're migrating may have already been initialized.

This part should be only in `reset-database`.

Reply:
If you delete it and create it from scratch, you should run `reset-database` to initialize it. `migrate` is only for migrating an already initialized database.

> When database already have to admin fixture, it is not loaded again.

But it will report an error which we do not want.

Reply:
If there's a case where you need to create a database without doing a reset, you need a separate job that initializes it (migrates it and loads fixtures).

Reply:
Ok. I delete load fixture from `migrate-database.yml` playbook. The solution you offered will be considered in other pull request.